pkg/scheduler/internal/cache/comparer/cache_comparer_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package factory
+package comparer
```

Comment:
Do these tests compile if you change this package name to `comparer_test`?

Reply:
It should be ok, but the package name `comparer_test` should not be the naming method recommended by kubernetes?

Reply:
em m m, maybe you should say the file name instead of the package name.

Reply:
I actually did mean the package name. It forces us to test the public API without doing any "mucking" with package internals. It's not always easy, but in this case I think it might be.